clients/cli/src/test/java/org/apache/gravitino/cli/TestMetalakeCommands.java

```diff
@@ -76,7 +78,7 @@ void testMetalakeDetailsCommand() {
                 mockCommandLine, mockOptions, CommandEntities.METALAKE, CommandActions.DETAILS));
     doReturn(mockDetails)
         .when(commandLine)
-        .newMetalakeDetails(GravitinoCommandLine.DEFAULT_URL, false, "metalake_demo");
+        .newMetalakeDetails(GravitinoCommandLine.DEFAULT_URL, false, "metalake_demo", "");
```

Comment:
If --output is not specified on teh CLI then its value should be null not an empty string?